manifests/istio-telemetry/prometheus/templates/configmap.yaml

```diff
@@ -84,8 +84,9 @@ data:
       relabel_configs:
       - source_labels: [__meta_kubernetes_service_name, __meta_kubernetes_endpoint_port_name]
         action: keep
-        regex: istio-pilot;http-monitoring
-
+        regex: istiod;http-monitoring
```

Comment:
as above, do we care about legacy deployment architectures ?

Reply:
This is master, so I don't think so - we only support Istiod on master now.